cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/GitPatAuthenticationStrategy.java

```diff
@@ -30,16 +30,23 @@
 /**
  * An {@link AuthenticationStrategy} to use with GitHub and Personal Access Tokens.
  */
-public class GitPATAuthenticationStrategy implements AuthenticationStrategy {
+public class GitPatAuthenticationStrategy implements AuthenticationStrategy {
 
   private static final String GITHUB_PAT_USERNAME = "oauth2";
   private final SecureStorePasswordProvider credentialsProvider;
 
-  public GitPATAuthenticationStrategy(SecureStore secureStore, RepositoryConfig config,
+  /**
+   * Construct a Git PAT auth strategy.
+   * 
+   * @param secureStore {@link SecureStore} to fetch the secrets with.
```

Comment:
checkstyle warning

Reply:
hmmm, isn't it already an empty line? Or should i even remove that `*` 🤔